### DISCUSSION TITLE
GSOC 2024 tentative project ideas

### DISCUSSION THREAD

**Discussion by parthkl021** [openvinotoolkit/openvino#21776] [Google Summer of Code]
Is there any tentative list of project ideas for GSOC 2024 ?
I found last year ideas quite interesting and wanted to know if there are any new projects so that I can start working on it .

**Comment by adrianboguszewski** [comment]
Hey @parthkl021. Unfortunately, there is no list for now, but we expect to have it ready in February. Stay tuned :) 
Meanwhile, please look at small projects here: https://github.com/orgs/openvinotoolkit/projects/3

**Comment by AishwaryaDekhane** [comment]
I am Aishwarya Dekhane, currently pursuing a Master's in AI, specializing in computer vision. My interests lie in computer vision, deep learning, and neural networks. Having followed OpenVINO extensively, I am keen to know whether there will be a publication of prerequisite tasks this year also, and if so, when?

**Reply by adrianboguszewski** [reply]
The first draft of the prerequisite task is already published here: https://github.com/openvinotoolkit/openvino/wiki/Google-Summer-Of-Code#prerequisite-task. However, it may change a little bit before an official announcement.
